lib/oracle.js

```diff
@@ -529,6 +588,6 @@ Oracle.prototype.ping = function (cb) {
   this.execute('select count(*) as result from user_tables', [], cb);
 };
 
-require('./migration')(Oracle);
-require('./discovery')(Oracle);
-require('./transaction')(Oracle);
+require('./migration')(Oracle, oracle);
+require('./discovery')(Oracle, oracle);
```

Comment:
Maybe i missed something here, I think only module `./transaction` take two parameters, is it fine to pass in `oracle` to `./discovery` and `./migration` as well?